### PR TITLE
Create DeleteTask command, DeleteTaskCommandParser 

### LINKED ISSUE #6: As a manager, I can remove tasks that were created.

### PR DESCRIPTION
**Pull request by Nicklelodeon**
Add necessary JUnit tests for DeleteTask, DeleteTaskCommandParser
Resolve #6

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public DeleteTaskCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model, OfficeConnectModel officeConnectModel) throws CommandException {
+        requireAllNonNull(model, officeConnectModel);
+        List<Task> lastShownList = officeConnectModel.getTaskModelManager().getFilteredItemList();
+
+        System.out.println(lastShownList.size());
```
Comment (cyiting) [inline src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java:38]: Please remove your debugging statements

File: src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java [hunk 2]
```diff
@@ -23,10 +23,8 @@ public class ListTaskCommandTest {
 
     @BeforeEach
     public void setUp() {
-        RepositoryModelManager<Task> sampleModelManager = new RepositoryModelManager<>(new Repository<Task>());
-        Task defaultTask = new TaskBuilder().build();
-        sampleModelManager.addItem(defaultTask);
-        model = new OfficeConnectModel(sampleModelManager, new RepositoryModelManager<>(new Repository<PersonTask>()));
+        model = new OfficeConnectModel(new RepositoryModelManager<>(getTypicalTaskRepository()),
```
Comment (cyiting) [inline src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java:26]: I like that made you use of `getTypicalTaskRepository()` in your test

**Review by cyiting — COMMENTED** [review]
LGTM! Except for the presence of the debugging statement.

**Review by cyiting — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java (hunk 1 above)
Comment (Nicklelodeon) [reply src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java]: oops ok removed it!

**Pull request merged**